app/main.py

```diff
@@ -16,4 +16,17 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+def sort_var(*args) -> dict:
+    return_list = {"mutable": [], "immutable": []}
+    for obj in args:
+        if isinstance(obj, (list, dict, set)):
+            return_list["mutable"].append(obj)
+        else:
+            return_list["immutable"].append(obj)
```

Comment:
The main purpose of this task is to remember which types are mutable and which are immutable in python, so you can just fill in variables in a dict mannualy

Reply:
Yes, but its not interesting.